move-mutator/tests/move-assets/simple/sources/StillSimple.move

```diff
@@ -39,11 +79,70 @@ module TestAccount::StillSimple {
         };
     }
 
+    #[test]
+    fun sample4_valid_usage_test() {
+        // Does this test even make any sense?
```

Comment:
The test itself? No 😄

Reply:
However, some mutants may generate overflows in that function for particular arguments.